.github/workflows/vib.yaml

```diff
@@ -16,15 +16,6 @@ env:
   CSP_API_TOKEN: ${{ secrets.CSP_API_TOKEN }}
   VIB_PUBLIC_URL: https://cp.bromelia.vmware.com
 jobs:
-  vib-validate: # make sure chart is linted/safe
```

Comment:
You need to remove the **needs** clause in #31 otherwise this workflow is going to fail as the vib-validate is now gone.

This also means the vib-k8s-verify job that runs for wordpress/redis will trigger immediately and will ask for approval ( as it uses the environment trick, i.e. the job is tied to an environment that needs approval ). 

I'm thinking that perhaps this workflow needs to be chained to the lint workflow that you created. Doing a quick search, it seems that this could be done with [workflow_run](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run).

Reply:
@mpermar I tried the `workflow_run` condition but it won't allow me to filter by paths changed. I tried adding pull_request_target but according to the documentation the multiple events condition is an OR, not AND, that means the workflow would be run in every PR regardless of the chart. Because of that, I'd leave it as it is.